chain-extensions/xvm/src/lib.rs

```diff
@@ -18,19 +18,15 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use astar_primitives::xvm::{CallError, Context, VmId, XvmCall};
+use astar_primitives::xvm::{Context, VmId, XvmCall};
 use frame_support::dispatch::Encode;
-use pallet_contracts::{
-    chain_extension::{ChainExtension, Environment, Ext, InitState, RetVal},
-    Origin,
-};
+use pallet_contracts::chain_extension::{ChainExtension, Environment, Ext, InitState, RetVal};
 use sp_runtime::DispatchError;
 use sp_std::marker::PhantomData;
 use xvm_chain_extension_types::{XvmCallArgs, XvmExecutionResult};
 
 enum XvmFuncId {
     Call,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The enum `XvmFuncId` is missing the `Call` variant. Please add it.
```diff
 enum XvmFuncId {
     Call,
    // TODO: expand with other calls too
 }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->